src/main/kotlin/org/rust/lang/core/crate/impl/CargoBasedCrate.kt

```diff
@@ -47,7 +47,9 @@ class CargoBasedCrate(
     override val cargoWorkspace: CargoWorkspace get() = cargoTarget.pkg.workspace
     override val kind: CargoWorkspace.TargetKind get() = cargoTarget.kind
 
-    override val cfgOptions: CfgOptions get() = cargoTarget.pkg.cfgOptions
+    override val cfgOptions: CfgOptions get() = cargoTarget.cfgOptions
+    override val evaluateUnknownCfgToFalse: Boolean
+        get() = cargoTarget.pkg.cfgOptions != null || cargoTarget.pkg.customBuildTarget == null
```

Comment:
This condition is not really obvious. Probably it should be separated into three cases, e.g. “no build.rs”, “evaluated build.rs”, and “ignored build.rs"